docs/6_logging.md

```diff
@@ -0,0 +1,67 @@
+# Table of Contents
+
+* [Logging in Fundus](#logging-in-fundus)
+  * [Principals](#principals)
+  * [Accessing loggers](#accessing-loggers)
+  * [Changing log levels](#changing-log-levels)
+  * [Format and handlers](#format-and-handlers)
+
+# Logging in Fundus
+
+This tutorial will introduce you to the logging mechanics used in Fundus
+
+## Principals
+
+Fundus uses module scoped logging with module names as logger names.
+Not every module has a logger per se, but every module that logs a message has.
+All module related implementation is centralized in Fundus' logging module under `fundus.logging`.
+Fundus uses 4 different log levels:
+- DEBUG: Not relevant to the average user and mainly used for debugging.
+- INFO: Could be interesting to the user, but not necessarily.
+- WARNING: Something went wrong, but we're trying to fix it.
+- ERROR: Either we tried or not even bothering to resolve this.
+With the default one being `ERROR`.
+
+*__NOTE__*: Depending on the spawn method (spawn) your OS uses to spawn new processes in python (this effects mostly Windows), log messages beneath `ERROR` won't be received when using multiprocessing. 
+
+## Accessing loggers
+
+You can import a specific logger from the corresponding module like this:
+
+````python
+from fundus.scraping.crawler import logger
+````
+
+Or find a collection of all existing loggers with their module names here:
+
+````python
+from fundus.logging import loggers
```

Comment:
I think it might be nice to add an example or overview of the module names here. Something along the lines of:

Fundus support logging in the follwoing modules: crawling, scraping...

If you want to access a certain logger, you would achieve this by
crawling_logger = loggers['crawling']

Reply:
Yeah, thought about that as well. The thing that kept me away was the maintenance effort. The added modules would be static, so every newly added logger has to be added by hand, which I'm 99% sure we will forget. So I thought it would be better to have no list at all rather than a potentially incomplete list, especially since the modules can still be viewed through the `loggers` variable. What du you think?

Reply:
I think my main concern was that it's a bit of extra work for the users to get access to the loggers names, or at least the format that is being used and with the given example this seems to be partially solved. I don't think though that an incomplete list is a problem as long as it's marked as such. But nevertheless, I think the example should be sufficient as well